content/blog/2022/10/2022-10-20-jenkins-election-announcement.adoc

```diff
@@ -0,0 +1,97 @@
+This is an opportunity for Jenkins contributors and community members to sign-up to vote in elections, and acknowledge other community members by nominating them for a position.
+The deadline for nominations and voter registration is *November 9th*.
+
+Participation in the election process requires an account on link:https://community.jenkins.io[community.jenkins.io].
+You can use your Github account or create a new discourse account for link:https://community.jenkins.io[community.jenkins.io].
+Additionally, you must be able to showcase at least one contribution done before the September 1st, 2022.
+As mentioned on link:/participate/[jenkins.io/participate], they are many different ways to contribute to Jenkins despite having different scopes.
+This makes it difficult to consistently measure contributions.
+Therefore, we are trusting participants and not requiring that you provide evidence of contribution as part of your voter registration.
+We reserve the right to ban the specific account from the election process if we identify abuse.
+
+The election has 3 stages:
+
+* Nomination and voter registration
+* Voting
+* Results announcement
+
+== Voters
+
+To be eligble to vote, we need to submit a list of email addresses to the link:https://civs.cs.cornell.edu[Condorcet Internet Voting Service].
+We ask that all community members that are interested in voting, and meet the requirements, join the group link:https://community.jenkins.io/g/election-voter-2022[election-voter-2022] on link:https://community.jenkins.io[community.jenkins.io].
+The group will be open to join during the registration period.
+
+== Nominations
+
+Also during this period, we invite community members to nominate candidates by sending a message to the link:https://community.jenkins.io/g/election-committee[election-committee] group, mentioning the position and the motivation.
+On November 9th, the nomination period will end.
+Nominees will be notified, and asked for confirmation that they are interested in running as a candidate.
+The list of candidates will be announced on November 14th.
+
+*Anyone can nominate candidates.*
+
+For the 2022 election, nominations are open for the following positions:
+
+* link:/project/governance/#governance-board[Board members]
+* link:/project/team-leads/#documentation[Documentation Officer]
+* link:/project/team-leads/#events[Events Officer]
+* link:/project/team-leads/#infrastructure[Infrastructure Officer]
+* link:/project/team-leads/#release[Release Officer]
+* link:/project/team-leads/#security[Security Officer]
+
+Additional information about the roles can be found on the link:/project/team-leads/[team leads] page.
+
+== Election
+
+On the 17th of November, candidates will be announced and everybody that registerd will be notified by email to vote using link:https://civs.cs.cornell.edu[civs.cs.cornell.edu].
+Once candidates have been annouinced, registration will be closed.
```

Comment:
It might be good to emphasis this no?

```suggestion
Once candidates have been announced, registration will be closed.
```

Reply:
thanks for catching that! I added some emphasis to it, so hopefully it stands out better